modules/@apostrophecms/schema/ui/apos/components/AposInputColor.vue

```diff
@@ -20,7 +20,7 @@
           >
             <Picker
               v-bind="pickerOptions"
-              :model-value="next"
+              :model-value="pickerValue"
```

Comment:
Unrelated to this PR.

Reply:
I think that was what @itsrogil and @ValJed found to fix the new warning.
Not directly related to the original ticket, but it still removes a console warning and fixes the color picker